src/opnsense/mvc/app/views/OPNsense/Firewall/alias.volt

```diff
@@ -633,10 +633,13 @@
                                                 </tr>
                                             </tbody>
                                         </table>
+                                        <div class="hidden" data-for="help_for_alias.frequency">
+                                                               <small>
+                                                {{lang._('The frequency that the list should be updated, where 1 would be daily, 7 would be weekly. Time is the time of day, e.g. 12:00 would be midday')}}
```

Comment:
I'm not 100% sure we should change it, how it works is days+time equals the validity of the alias, so, if you're choosing 1+12 that would mean every 36 hours it would refresh. 

The mechanism is more comparable to TTL in that regard, hence it translates to alias ttl in our template 
https://github.com/opnsense/core/blob/ae5692b477db1701491bce55db5317b0a73728c0/src/opnsense/service/templates/OPNsense/Filter/filter_tables.conf#L27

So the question is, should we change the label, and if so, what would the phrase most people will understand.

Reply:
So it doesn't go and refresh the list then?

Reply:
it refreshes the list after that amount of time, not on a specific time (12.00)

Reply:
So change it to 'Refresh Frequency and change the help to - 'The alias will be refreshed after the given time has expired', or leave it as it is and update the Wiki as it doesn't match what's in the GUI - one or the other.

Reply:
I'm ok with that, good to have this discussion.

Reply:
:) Yes, but which, change the GUI or Wiki or both?

Reply:
as you proposed it, I would expect both :) The help text triggered me, I can probably live with most variations, as long as it's more clear for others what we mean... the help text pointed to a misunderstanding.

Reply:
OKey dokey.. will mod accordingly..